gramex/handlers/mlhandler.py

```diff
@@ -40,6 +43,7 @@
     'nums': [],
     'cats': [],
     'target_col': None,
+    'CV': True,
```

Comment:
Make it lowercase.

Reply:
We have to support three cases for the `cv` option:

- [ ] If the user sets `cv: false` - then no cross validation happens
- [ ] If the user sets `cv: 4` (or some other integer) pass it straight to `cross_val_score`
- [ ] The default should be `cv: None`, and in this case, the user should not have to write anything in gramex.yaml